include/aie/Dialect/AIE/IR/AIETargetModel.h

```diff
@@ -53,8 +53,12 @@ using TileID = struct TileID {
 };
 
 class AIETargetModel {
+  bool virtualized;
+
 public:
-  AIETargetModel() = default;
+  AIETargetModel(bool virtualized = true) : virtualized(virtualized) {}
```

Comment:
This seems insufficient, since we actually have many potential 'virtual' targets (for instance, of different column widths.